cmd/crc/cmd/daemon.go

```diff
@@ -129,7 +131,15 @@ func run(configuration *types.Configuration, endpoints []string) error {
 			}
 		}()
 	}
-	return <-errCh
+
+	c := make(chan os.Signal, 1)
+	signal.Notify(c, os.Interrupt, syscall.SIGTERM)
```

Comment:
My understanding of https://golang.org/pkg/os/signal/#example_Notify is that on Windows this will also behave properly?

Reply:
yes!